packages/frontend2/src/app/(legacy)/faq/page.tsx

```diff
@@ -0,0 +1,88 @@
+import { isArray } from 'lodash'
+import { type Metadata } from 'next'
+import { ContentWrapper } from '~/app/_components/content-wrapper'
+import { Markdown } from '~/app/_components/markdown'
+import { ScrollToTopButton } from '~/app/_components/scroll-to-top-button'
+import { faqItems } from './faq-items'
+
+export const metadata: Metadata = {
+  title: 'FAQ - L2BEAT',
+  description:
+    'Frequently Asked Questions about L2BEAT - an analytics and research website about Ethereum layer 2 scaling.',
+  openGraph: {
+    url: '/faq',
+  },
+}
+
+export default async function Page() {
+  return (
+    <>
+      <Header />
+      <ContentWrapper>
+        <div className="flex">
+          <div className="mt-12 hidden w-72 shrink-0 flex-col gap-4 lg:flex">
+            {faqItems.map((item) => (
+              <a
+                key={questionToId(item.question)}
```

Comment:
It's more efficient to map the array once to generate the ids than to use `questionToId` 5 times inside one file 😛